tests/Aspire.Hosting.Azure.Tests/AzureEnvironmentResourceTests.cs

```diff
@@ -183,6 +184,41 @@ private sealed class TestProject : IProjectMetadata
         public LaunchSettings? LaunchSettings { get; set; }
     }
 
+    [Fact]
+    public void AzurePublishingContext_IgnoresAzureBicepResourcesWithIgnoreAnnotation()
+    {
+        // Arrange
+        using var builder = TestDistributedApplicationBuilder.Create(DistributedApplicationOperation.Publish);
```

Comment:
Create the builder similar to how the AzurePublishingContext_CapturesParametersAndOutputsCorrectly_WithSnapshot does, run the application and use Verify to verify the azure resources in the snapshot.